payloads/external/Makefile.inc

```diff
@@ -235,12 +243,12 @@ img/setup-compression := $(CBFS_PAYLOAD_COMPRESS_FLAG)
 
 # sgabios
 
-payloads/external/sgabios/sgabios.bin sgabios:
-	$(MAKE) -C payloads/external/sgabios
+# payloads/external/sgabios/sgabios.bin sgabios:
+#	$(MAKE) -C payloads/external/sgabios
 
-cbfs-files-y += vgaroms/sgabios.bin
-vgaroms/sgabios.bin-file := payloads/external/sgabios/sgabios.bin
-vgaroms/sgabios.bin-type := raw
+# cbfs-files-y += vgaroms/sgabios.bin
+# vgaroms/sgabios.bin-file := payloads/external/sgabios/sgabios.bin
+# vgaroms/sgabios.bin-type := raw
```

Comment:
@miczyg1 why we just comment ?

Reply:
When commited this change, was thinking about leaving it for future, in case we would like to bring sgabios back. If not necessary I can remove that completely.

Reply:
@miczyg1 I believe git history should be enough, so we don't need it in code.